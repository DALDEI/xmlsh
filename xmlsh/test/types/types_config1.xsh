# Config modules
import c=types.config
import p=types.properties


readconfig -format ini -file ../../samples/types/config.ini _c

s=c:section($_c params)
echo ${s[param1]}
echo c:sections( $_c )
echo c:value( $_c params.param2 )
s=c:section($_c section)
echo p:get( $s name )
echo p:get( $s web )

echo ${_c[section.name]}
echo ${_c[section.var1]}