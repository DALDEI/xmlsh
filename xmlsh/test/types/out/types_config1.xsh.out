parameter 1
aws default params
parameter 2
